### ISSUE TITLE
🐛 Bug: Insert new row for tables with auto generated primary key

### ISSUE THREAD

**Issue by mertmit** [nocodb/nocodb#4029]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Observed in at imported tables. We use `ncRecordId` as auto-generated id but it is marked as required so you can't create a column if you don't manually insert an ID.

Cause: packages/nc-gui/composables/useViewData.ts > insertRow function

### Desired Behavior

We should be able to create a new record without providing auto-generated columns just like auto-incremented column.

### Project Details

-

**Comment by pranavxc** [comment]
Fixed in `0.98.1`

**Comment by pedralmeida22** [comment]
The new row is created with with auto generated primary key but is returning:
```
{
    "msg": "Undefined binding(s) detected when compiling SELECT. Undefined column(s): [ncRecordId]
    ...
}
```
